app/routes.py

```diff
@@ -1,2 +1,277 @@
 from flask import Blueprint
+import flask_migrate
+from app import db 
+from app.models.task import Task 
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response 
+from flask import jsonify
+from datetime import date
+import requests
+import os
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@tasks_bp.route("", methods=["POST", "GET"], strict_slashes=False)
+def handle_task():
+    if request.method == "GET":  
+        query_param_value = request.args.get("sort")
+        if query_param_value == "asc":
+            tasks = Task.query.order_by(Task.title.asc())
+
+        elif query_param_value == "desc":
+            tasks = Task.query.order_by(Task.title.desc())
+
+        else:
+            tasks = Task.query.all()
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": False
+            })
```

Comment:
Why not use `to_json` here? This passes all the tests but only because the tests don't test the case where get all tasks is called after a task has been marked complete. 
In other words, it doesn't respond accurately if any of the tasks are actually complete.